### DISCUSSION TITLE
ViewLocator causing errors with 11.0

### DISCUSSION THREAD

**Discussion by cristinathoughtpennies** [AvaloniaUI/Avalonia#9951] [Q&A] [answered]
Hello!
I am currently on the latest nightly build and the ViewLocator class is causing compiler errors. Since it's a templated class, I'm wondering if there is a correct version of the class that I should be using instead?

**Comment by maxkatz6** [comment, marked as answer]
What kind of compile errors?
I couple of hours ago has tested latest templates with various build options and seen no compile errors.
Can you provide more information?

**Reply by cristinathoughtpennies** [reply]
Yes! Sorry looking back at what I wrote, it's very vague...
So I moved from 0.10.18 to latest, and there were a couple issues. For example, references to IControl needed to be Control instead. I resolved them, but I couldn't find a version of ViewLocator in the latest repo that didn't have the issues, but it sounds like it exists, and I just couldn't find it.

**Reply by maxkatz6** [reply]
Oh, I see. Just noticed that you are using latest nightly.
Templates weren't yet updated to use latest nightly. They will be updated only when preview5 is released in next week or two.

**Reply by maxkatz6** [reply]
But you can get it work by replacing `IControl` with `Control`.
See https://github.com/AvaloniaUI/Avalonia/wiki/Breaking-Changes#010--110
